doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
```

Comment:
I feel we should do it now itself... (mv to `search.rst` I feel?)... Sorry, I overlooked that!!